native-embedding/src/test/c/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+cmake_minimum_required(VERSION 3.22)
+project(native-embedding-test)
+set(CMAKE_C_STANDARD 11)
+set(CMAKE_CONFIGURATION_TYPES Release)
+set(CMAKE_BUILD_TYPE Release)
+if(NOT WIN32)
+    set(C_FLAGS "-fPIC -g")
+    set(CMAKE_C_FLAGS ${C_FLAGS})
+endif()
+# Fix Windows paths
+STRING(REGEX REPLACE "\\\\" "/" PROJECT_MAIN_DIR_FIXED $ENV{PROJECT_MAIN_DIR})
+STRING(REGEX REPLACE "\\\\" "/" JAVA_HOME_FIXED $ENV{JAVA_HOME})
+STRING(REGEX REPLACE "\\\\" "/" LIB_API_PATH_FIXED $ENV{LIB_API_PATH})
+# Set paths
+set(HELPER_LIB_SRC_DIR ${PROJECT_MAIN_DIR_FIXED}/src/main/c)
+set(TEST_SRC_DIR ${PROJECT_MAIN_DIR_FIXED}/src/test/c)
+set(LIB_API_PATH ${LIB_API_PATH_FIXED})
+set(JNI_INCLUDE_DIR ${JAVA_HOME_FIXED}/include)
+set(JNI_PLATFORM_INCLUDE_DIR ${JAVA_HOME_FIXED}/include/$ENV{PLATFORM_NAME})
+# Set API library name
+set(APILIB $ENV{LIB_API_NAME})
```

Comment:
This could use a bit more explanation. `# Set API library name`, right, I can see that, that's just repeating the information already in the code. Is this the native-image of `API.java`?